test/cli/suggest_t_unsafe/suggest_t_unsafe.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+tmp="$(mktemp -d)"
+infile="test/cli/suggest_t_unsafe/suggest_t_unsafe.rb"
+cp "$infile" "$tmp"
+
+cwd="$(pwd)"
+cd "$tmp" || exit 1
+
+if "$cwd/main/sorbet" --silence-dev-message --suggest-unsafe -a suggest_t_unsafe.rb 2>&1; then
```

Comment:
Could you add a test for a non-default value of `--suggest-unsafe` as well?

Reply:
Ah i knew you were gonna ask for this i was trying to sneak it past you 😅

Reply:
1fb89b2